routes/city.js

```diff
@@ -309,10 +309,12 @@ router.post("/spec-search", tokenAuthentication, async (req, res) => {
     
     for(var i = 0; i < disID.length; i++) {
       if ( i==0 ) {
-        exitData = filteredSearch.filter(function(city) {
```

Comment:
Why need to delete filter function??
Without filtering we will lose the whole feature.

Reply:
Re-updated

Reply:
This is not my fault. @IslaMcN  had revome it and merge it into a contest. Anyway, I had fixed it.
https://github.com/Lambda-School-Labs/best-places-to-live-be/pull/153/files